docs/OpenSSL300Design.md

```diff
@@ -2864,11 +2888,23 @@ Security Policy statement regarding the <a href="https://csrc.nist.gov/publicati
   <tr>
    <td>
    </td>
+  <tr>
```

Comment:
You seem to be inserting this hunk at a wrong place

Reply:
Good point.  Editing this file is nasty.

Reply:
Fixed now.